deploy/examples/create-external-cluster-resources.py

```diff
@@ -1122,6 +1122,41 @@ def get_ceph_dashboard_link(self):
         return json_out["dashboard"]
 
     def create_rgw_admin_ops_user(self):
+        # If the user already exists, we query it
+        result = self._query_user_info()
+        if result is not None:
+            return result
```

Comment:
CI is passing 
But I see a lot of mistakes with this,
I think we need to make CI more robust and check more cases.

1) First of all we don't have to return single argument to parent call
This is the final output
```
return (
            jsonoutput["keys"][0]["access_key"],
            jsonoutput["keys"][0]["secret_key"],
            info_cap_supported,
            "",
        )
```

2) Secondly adding of info cap can be improved,
But for now I would say, if the user already exists but it folow the condition we should call that info cap addition,

SO info cap addition would be called from the parent function only.

Please try to check each and every case,
You can use this CI for testing if needed, I fail it forcefully so we can test the changes, https://github.com/rook/rook/pull/12162

Reply:
@Sheetalpamecha ^^

Reply:
Hi Parth, 
I have been trying to setup the external cluster and running this locally to test and try out more cases and that's why its taking a little more time.

> You can use this CI for testing if needed, I fail it forcefully so we can test the changes, https://github.com/rook/rook/pull/12162

I couldn't push the changes here, due to access issues.

Reply:
Setting up it locally would be a difficult process, as you need a standalone cephcluster,

I just failed this CI https://github.com/rook/rook/actions/runs/5200425621/jobs/9515629185 you can test the changes here with different permutations.

>I couldn't push the changes here, due to access issues.

No problem, thanks for taking the look:)

Reply:
Yes I have been running into quite a few issues with manual setup.
Thanks for suggesting this. I happen to test some cases on such failing CI.

PTAL at the new modified code.